### PR TITLE
HODS-313 | Update API documentation for scope handler pattern based authorisation

### PR DESCRIPTION
**Pull request by ajchilts**

### CODE REVIEW

File: resources/public/api/conf/1.0/docs/post-corporation-tax-description.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<p>Use this endpoint to match details about an organisation against HMRC’s records.</p>
+<p>Details include: company registration number (CRN), employer name, and address. On a successful match, a matchId is returned.</p>
```
Comment (owen-opencast) [inline resources/public/api/conf/1.0/docs/post-corporation-tax-description.md:2]: Move "On a successful match..." to the third paragraph, to match the SA description?

**Review by owen-opencast — CHANGES_REQUESTED** [review]

**Review by Luka-Abey — APPROVED** [review]

**Review by owen-opencast — APPROVED** [review]

File: resources/public/api/conf/1.0/application.raml [hunk 2]
```diff
@@ -75,13 +75,13 @@ uses:
                     description: "The organisation's details provided did not match with HMRC's records"
                     value:
                       code: MATCHING_FAILED
-  /self-assessment:
+  /matching/self-assessment:
```
Comment (Luka-Abey) [inline resources/public/api/conf/1.0/application.raml:78]: matching is duplicated in the full path 
```suggestion
  /self-assessment:
```

File: resources/public/api/conf/1.0/application.raml [hunk 3]
```diff
@@ -166,10 +166,10 @@ uses:
           example: 57072660-1df9-4aeb-b4ea-cd2d7f96e430
       get:
         displayName: "Get links to Self Assessment and number of employees details for a matched organisation"
-        description: "This endpoint takes a matchId and returns matched organisation details. It also provides links to get Self Assessment and number of employees details for a matched organisation."
-        is: [headers.acceptHeader]
-        (annotations.scope): "read:organisations-matching"
-        securedBy: [ sec.oauth_2_0: { scopes: [ "read:organisations-matching" ] } ]
+        description: !include docs/get-self-assessment-tax-description.md
+        is: [headers.acceptHeader, headers.contentHeader]
```
Comment (Luka-Abey) [inline resources/public/api/conf/1.0/application.raml:170]: should not include content-type for get endpoint 
```suggestion
        is: [headers.acceptHeader]
```

**Review by Luka-Abey — CHANGES_REQUESTED** [review]
couple small bits on re-review

**Review by Luka-Abey — APPROVED** [review]

**Pull request merged**